Cargo.lock

```diff
@@ -137,6 +138,18 @@ dependencies = [
  "syn",
 ]
 
+[[package]]
+name = "bdk-reserves"
+version = "0.14.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "81e5def53134305f5c44dfd5b1b94609fc194d983d0b816e5ca7635d44a9d6dd"
+dependencies = [
+ "base64 0.11.0",
+ "bdk",
+ "bitcoinconsensus",
```

Comment:
`bdk` has a `verify` feature that uses `bitcoinconsensus`. Would there a be a way in future to use `bdk::verify` instead of `bitcoinconsensus` in the `bdk-reserves` dependency?